src/python/CMakeLists.txt

```diff
@@ -38,12 +38,33 @@ execute_process (
 file (TO_CMAKE_PATH "${PYTHON_PREFIX}" PYTHON_PREFIX)
 execute_process (
    COMMAND ${PYTHON_EXECUTABLE} -c
-       "import site, sys; sys.stdout.write(site.getusersitepackages().replace(site.getuserbase(), site.PREFIXES[-1]))"
+       "	\n
+import site, sys \n
+dirs = site.getsitepackages(); \n
+dir = dirs[0] \n
+for d in dirs: \n
+	if d.count(\"packages\"): \n
+		dir = d[d.index(\"packages\")-5:] \n
+		break \n
+py = \"${PYTHON_EXECUTABLE}\" \n
+py = py[py.rfind(\"/\")+1:] \n
+prefix = \"${PYTHON_PREFIX}\"[\"${PYTHON_PREFIX}\".rfind(\"/\"):] \n
+dir = \"{0}/lib/{1}/{2}\".format(prefix, py, dir) \n
```

Comment:
`py` will not work on e.g. Edison - where the executable is `/usr/bin/python2`, but the packages dir is `/usr/lib/python2.7/site-packages`, i.e. Python executable name is not equal to `/usr/lib` subfolder.